_layouts/default.html

```diff
@@ -53,7 +53,7 @@
     <p class="availability">
       Available in <a target="_blank" href="https://www.chromestatus.com/feature/{{ page.feature_id }}">Chrome {{ page.chrome_version }}+</a> |
       <a target="_blank" href="{{ site.gh_url_prefix }}{{ page.url | remove_first: 'index.html' }}">View on GitHub</a> |
-      <a target="_blank" href="https://www.chromestatus.com/samples">Browse Samples</a>
+      <a {% if page.index %}{% else %}target="_blank"{% endif %} href="{% if page.index %}{{ page.index }}{% else %}https://www.chromestatus.com/samples{% endif %}">Browse Samples</a>
```

Comment:
There is no "NOT" operator in Liquid templates. https://github.com/Shopify/liquid/wiki/Liquid-for-Designers#if--else

Here's a new patch. WDYT?